src/components/Utilities/TypographyUsage.css

```diff
@@ -44,6 +44,10 @@
   @mixin eds-typography-preset-009-bold;
 }
 
+.u-theme-typography-body-text-xl {
+  @mixin eds-typography-preset-002;
+}
+
```

Comment:
just checking design is aware of this & is gonna update their presets?

Reply:
Nah not yet, have to sync about this so will mention it then

Reply:
Update: syncing with Sean to get the ok for some of the mock differences and regarding this preset

Reply:
Update: synced with Sean, he'll create an action item on the design side regarding this token